### ISSUE TITLE
Define standard metadata for parameters

### ISSUE THREAD

**Issue by MaxGhenis** [UBICenter/openfisca-us#57]
e.g. unit, reference

**Comment by MaxGhenis** [comment]
* unit: /1 or currency-USD
* reference:
* period:

**Comment by martinholmer** [comment]
@MaxGhenis, Aren't the items you suggest [here](https://github.com/PolicyEngine/openfisca-us/issues/57#issuecomment-907668623) in issue #57 already in parameter YAML files?  

Not sure what you are suggesting.  Do you want some kind of documentation for developers that says what the minimal metadata for parameters should be?

**Comment by MaxGhenis** [comment]
> Do you want some kind of documentation for developers that says what the minimal metadata for parameters should be?

Yes, updated the title accordingly. Thanks for the documentation tag.

**Comment by DTrim99** [comment]
@MaxGhenis Closing as obsolete — the standard-parameter-metadata documentation ask targeted the repo wiki, which is abandoned; parameter conventions now live in the docs site. Reopen if a current-docs page is wanted.
